Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
`enumerize` is already included in the Gemfile

https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23